libs/Microsoft.MixedReality.WebRTC/Interop/ObjectInterop.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.MixedReality.WebRTC.Interop
     /// <summary>
     /// Handle to a native object.
     /// </summary>
-    internal abstract class ObjectHandle : SafeHandle
```

Comment:
This and `RefCountedObjectHandle` can go back to be internal, AFAICS we are exposing the `IntPtr` directly in `RemoteVideoTrack` so this is unneeded.

Reply:
changed to internal